spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -151,6 +151,101 @@
     end
   end
 
+  describe 'POST publish' do
```

Comment:
Same with this. Move to its own file

Reply:
And likewise, here I'd really like to see:

```
spec/requests/api/v2/providers/courses/publish_spec.rb
```

Even more so than for the model ... there's likely to be more tests here. :)

Reply:
Done. Thanks for the feedback!